src/main/groovy/org/transmartproject/batch/support/ScientificNotationFormat.groovy

```diff
@@ -0,0 +1,23 @@
+package org.transmartproject.batch.support
+
+import java.text.DecimalFormat
+import java.text.NumberFormat
+import java.text.ParsePosition
+
+/**
+ * Supports parsing of the scientific format numbers.
+ * Variations like 1.002e+9 and 1.002*10^9 are allowed.
+ */
+class ScientificNotationFormat extends NumberFormat {
+
+    @Delegate DecimalFormat decimalFormat = new DecimalFormat("0.###E0")
+
+    Number parse(String text, ParsePosition pos) {
+        String processedText = text
+                .replace('e', 'E')
+                .replace('E+', 'E')
```

Comment:
@cataphract It's usual thing to have negative exponents.  `DecimalFormat ` works fine with them. So I just rely on it here. Explicit plus sign is not parsed correctly. `DecimalFormat ` gives you back `1` for `1E+10`.